features/apim-analytics-feature/org.wso2.analytics.solutions.apim.analytics.feature/src/main/resources/siddhi-files/APIM_ACCESS_SUMMARY.siddhi

```diff
@@ -25,8 +25,19 @@ define stream ApiUsagePerVersionFilteredStream (apiName string, apiVersion strin
 
 define stream ApiUserAgentFilteredStream(apiName string, apiVersion string, apiContext string, apiCreator string, apiCreatorTenantDomain string, operatingSystem string, browser string, requestTimestamp long, gatewayType string, label string);
 
+define stream geoTempStream(apiName string, apiVersion string,apiContext string, apiCreator string, apiCreatorTenantDomain string, username string, userTenantDomain string, country string, city string, totalCount long, requestTimestamp long);
```

Comment:
can we make the formatting consistent? There are spaces after the ',' character, but not for some. Please check this in other files as well.

Reply:
Fixed in a21aa920ceb5fdbd3f34fa9dddb75b5630876048 of https://github.com/wso2/analytics-apim.